### ISSUE TITLE
PHP 7.2 Incompatible

### ISSUE THREAD

**Issue by PhilippeGauthier** [imgix/imgix-statamic#13]
Seeing errors because 

```
ErrorException in ImgixTags.php line 24:
The each() function is deprecated. This message will be suppressed on further calls
```

Looks like it needs to be updated.

**Comment by philliphartin** [comment]
I created a fix for this in https://github.com/imgix/imgix-statamic/pull/14 but it has yet to be accepted and merged.

**Comment by sherwinski** [comment]
Closed in #14
